src/Codeception/Module/REST.php

```diff
@@ -498,7 +498,13 @@ protected function execute($method, $url, $parameters = [], $files = [])
             $this->debugSection("Request", "$method $url " . $requestData);
             $this->response = (string)$this->connectionModule->_request($method, $url, [], $files, [], $parameters);
         }
-        $this->debugSection("Response", $this->response);
+        $response = $this->response;
```

Comment:
could you name as `printedResponse` so it would be obvious that this is can't be used as real response